dash/types/src/pairql/shared.ts

```diff
@@ -34,13 +34,21 @@ export interface BlockedApp {
   schedule?: RuleSchedule;
 }
 
+export type ChildComputerStatus =
```

Comment:
this is the new enum that the API sends back. in a few months, when we push everyone past 2.7.0, the dates will no longer be optional. but for now we'll have to support both types of data, since < 2.7.0 macapps don't send the richer detaill.